test/test_selector_test.rb

```diff
@@ -7,7 +7,8 @@ def setup
       target_sha = @repo.branches['change_fizz'].target.oid
       master_sha = @repo.branches['master'].target.oid
       base_sha = @repo.merge_base(target_sha, master_sha)
-      @selector = TTNT::TestSelector.new(@repo, target_sha, base_sha)
+      @testtask = TTNT::TestTask.get_instance('test')
```

Comment:
Is the `#get_instance` method really useful ? Can't we just define a dummy `TTNT::TestTask` object here with a new `Rake::TestTask` like:

~~~ruby
rake_task = Rake::TestTask.new do |t|
  t.pattern = "test/**/*_test.rb"
end

@testtask = TTNT::TestTask.new(rake_task)
~~~